### ISSUE TITLE
Keep getting couldn't connect to node

### ISSUE THREAD

**Issue by shogun0101** [ethereum/mist#2283]
#### System information
``````
Version: 0.8.10 Ethereum Wallet 64 bit
OS & Version: Windows 7 Professional
Node type: `eth/geth(default)`
``````
Unable to get Ethereum wallet to work on my PC and it used to work. Keeps saying couldn't connect to node? See the logs for more. 

![image](https://cloud.githubusercontent.com/assets/28785384/26213126/7e211a00-3bbd-11e7-9db1-d82647cebd59.png)

Here is what I found in node files. I am unable to attach them as a zip for some reason:
node.txt (see below)
WARN [05-18|10:26:29] No etherbase set and no accounts found as default 
INFO [05-18|10:26:29] Starting peer-to-peer node               instance=Geth/v1.6.0-stable-facc47cb/windows-amd64/go1.8.1
INFO [05-18|10:26:29] Allocated cache and file handles         database=C:\\Users\\smelvin\\AppData\\Roaming\\Ethereum\\geth\\chaindata cache=1024 handles=1024
INFO [05-18|10:26:30] Initialised chain configuration          config="{ChainID: 1 Homestead: 1150000 DAO: 1920000 DAOSupport: true EIP150: 2463000 EIP155: 2675000 EIP158: 2675000 Engine: ethash}"
INFO [05-18|10:26:30] Disk storage enabled for ethash caches   dir=C:\\Users\\smelvin\\AppData\\Roaming\\Ethereum\\geth\\ethash count=3
INFO [05-18|10:26:30] Disk storage enabled for ethash DAGs     dir=C:\\Users\\smelvin\\AppData\\Ethash                          count=2
INFO [05-18|10:26:30] Initialising Ethereum protocol           versions="[63 62]" network=1
INFO [05-18|10:26:30] Loaded most recent local header          number=0 hash=d4e567…cb8fa3 td=17179869184
INFO [05-18|10:26:30] Loaded most recent local full block      number=0 hash=d4e567…cb8fa3 td=17179869184
INFO [05-18|10:26:30] Loaded most recent local fast block      number=0 hash=d4e567…cb8fa3 td=17179869184
INFO [05-18|10:26:30] Starting P2P networking 

node.log.0 (see below)
WARN [05-18|10:25:42] No etherbase set and no accounts found as default 
INFO [05-18|10:25:42] Starting peer-to-peer node               instance=Geth/v1.6.0-stable-facc47cb/windows-amd64/go1.8.1
INFO [05-18|10:25:42] Allocated cache and file handles         database=C:\\Users\\smelvin\\AppData\\Roaming\\Ethereum\\geth\\chaindata cache=1024 handles=1024
INFO [05-18|10:25:43] Writing default main-net genesis block 
INFO [05-18|10:25:43] Initialised chain configuration          config="{ChainID: 1 Homestead: 1150000 DAO: 1920000 DAOSupport: true EIP150: 2463000 EIP155: 2675000 EIP158: 2675000 Engine: ethash}"
INFO [05-18|10:25:43] Disk storage enabled for ethash caches   dir=C:\\Users\\smelvin\\AppData\\Roaming\\Ethereum\\geth\\ethash count=3
INFO [05-18|10:25:43] Disk storage enabled for ethash DAGs     dir=C:\\Users\\smelvin\\AppData\\Ethash                          count=2
WARN [05-18|10:25:43] Upgrading db log bloom bins 
INFO [05-18|10:25:43] Bloom-bin upgrade completed              elapsed=0s
INFO [05-18|10:25:43] Initialising Ethereum protocol           versions="[63 62]" network=1
INFO [05-18|10:25:43] Loaded most recent local header          number=0 hash=d4e567…cb8fa3 td=17179869184
INFO [05-18|10:25:43] Loaded most recent local full block      number=0 hash=d4e567…cb8fa3 td=17179869184
INFO [05-18|10:25:43] Loaded most recent local fast block      number=0 hash=d4e567…cb8fa3 td=17179869184
INFO [05-18|10:25:43] Starting P2P networking

**Comment by shogun0101** [comment]
Bump. I still can't get this to work. What do I need to do?

**Comment by shogun0101** [comment]
Bump again. 8 days and no answers.

**Comment by DerekJarvis** [comment]
I'm having this same issue. Was this resolved?

**Comment by shogun0101** [comment]
No. Its a bug with geth. I used parity instead of geth to do the block chain. There is a video on youtube. https://www.youtube.com/watch?v=zOo3_aSXoZE&t=1s
